openlibrary/macros/CreateList.html

```diff
@@ -0,0 +1,34 @@
+<div class="hidden">
```

Comment:
Wrap this in a `$if render_once('CreateListModal'):`. We only need this HTML rendered once on a given page, since it can be re-used!